test/system/authors_test.rb

```diff
@@ -0,0 +1,21 @@
+require "application_system_test_case"
+
+class AuthorsTest < ApplicationSystemTestCase
+  attr_reader :author
+
+  def setup
+    @author = authors(:john_doe)
+  end
+
+  test "visiting the index" do
```

Comment:
_nitpick_ _suggestion_ 

Usually the test should describe the outcome and not the process. 

```suggestion
  test "should display the title" do
```